### PR TITLE
Apple website clone branch

### PR DESCRIPTION
**Pull request by brenoxav**
We made a clone of an old version of the Apple website webpage.
This project consists of building a webpage using:
- Images as a background;
- Gradients to give navbar a 3D effect;
- Grid to align the aside images;
- Styled form input;

### CODE REVIEW

File: index.html
Comment (teekaytech) [inline index.html]: Each of these items should be linked. You should change them to something like:

```
<li class="gradient hover">
  <a href="">Store</a>
</li>
```

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__This is an awesome project. I'm impressed by all the little details that you paid attention to. However, there are some changes that are required to be fixed before this project can be approved. Let's check them out.__


**[ REQUIRED CHANGES ]**

- [x] There are some issues with your README file.
- You have a nice title on your readme, but the description is not descriptive enough. Kindly describe more details about your project.
- The link to the screenshot of the project is broken, therefore the image is not rendering. Kindly fix it.
- Your live demo link is not working. You can try [this platform](https://raw.githack.com/) to get a working link.

- [x] Your background image does not cover the whole width of the page. 
- You added the `max-width` rule to the `<session>` and `<aside>` containers in your page. This is causing some white spaces to the left and right side of your page. Kindly remove those white spaces.




**Please make the required changes and submit another review request when you are done.**

* * *

**_You can reach out to me on twitter (@ola_lere), Github(@teekaytech) or slack (@taofeekolalere )_**
__Happy Coding :smiley:__

File: index.html
Comment (brenoxav) [reply index.html]: Added anchor tags to each item.

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Hello team! 😃 

Great job with the implemented changes! 👏 
But we still have some small changes to do before getting the project approved ⏩ 
Kindly check my comments below 👇  apply the changes and resubmit the project for  a new code review

## Changes
- Kindly modify the position of the navbar, now does not match with the original
- Kindly, reduce the height of the main section, you should be able to see the aside section on full screen
original: 
Capture
your project:
Capture1

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 

### Congratulations team, this project has passed all the mandatory requirements. It is therefore approved :heavy_check_mark:


**Do not forget to fill the project completion form.
Again, congratulations, you can merge to master.
Good luck with your next project.**

**I'm @taofeekolalere on slack.
Happy Coding :man_technologist:**

**Pull request merged**
